.buildkite/pipeline_perf.py

```diff
@@ -114,6 +114,10 @@ def build_group(test):
             {"exit_status": -1, "limit": 1},
         ]
     }
+    if REVISION_A:
+        # Enable automatic retries (up to 3 runs in total) and disable manual retries to suppress spurious issues.
+        test_data["retry"]["automatic"].append({"exit_status": 1, "limit": 2})
```

Comment:
I thought we said just 1 retry (e.g. 2 runs) :thinking: 

```suggestion
        # Enable automatic retries (up to 2 runs in total) and disable manual retries to suppress spurious issues.
        test_data["retry"]["automatic"].append({"exit_status": 1, "limit": 1})
```

Reply:
you're right! fixed.